lib/copy-sync/__tests__/copy-sync-prevent-copying-into-itself.test.js

```diff
@@ -60,59 +60,59 @@ describe('+ copySync() - prevent copying into itself', () => {
 
   describe('> when source is a directory', () => {
     describe('>> when dest is a directory', () => {
-      it(`of not itself`, () => {
+      it('of not itself', () => {
         const dest = path.join(TEST_DIR, src.replace(/^\w:/, ''))
         return testSuccess(src, dest)
       })
-      it(`of itself`, () => {
+      it('of itself', () => {
         const dest = path.join(src, 'dest')
         return testError(src, dest)
       })
-      it(`should copy the directory successfully when dest is 'src_dest'`, () => {
+      it('should copy the directory successfully when dest is \'src_dest\'', () => {
```

Comment:
Does the new standard version allow this?

```suggestion
      it("should copy the directory successfully when dest is 'src_dest'", () => {
```

Reply:
Thank you for the review.
No error with double quotes.
it changed.

Reply:
OK, I realize this is a ton of work, but there's quite a few instances of this. I very much dislike backslash-escaped quotes `\'`, if we could use double-quotes there instead? If you don't want to do it, I'll get to it sooner or later.

Reply:
Changed to double quotes.